panopticum/templates/page/component.html

```diff
@@ -1,6 +1,6 @@
 {% extends "layout/_base.html" %}
 
-{% block title %} [[ component.name ]] {% endblock title %}
+{% block title %} Component view {% endblock title %}
```

Comment:
That code fragment not in ```vue-component``` HTML element and can be managed by vue.js. Will be reverted at ```component.name``` after SPA migration